Update instructions for installing PySyft in the syft.js repo

## Where?
I would like to update the README for syft.js.

## Who?
The updated README will help users installing syft.js on machines running Ubuntu 18.04.4 LTS.

## What?
When I try to install PySfyt, I cannot complete the command `pip install .`.

## Additional Context
The error I get is:
```Building wheels for collected packages: syft, psutil
  Building wheel for syft (PEP 517) ... done
  Created wheel for syft: filename=syft-0.2.6-py3-none-any.whl size=390428 sha256=7d374660ed2580e3f6e34ff0b301bb3852dbe660b75f4c7b8ff2448dbaa46865
  Stored in directory: /tmp/pip-ephem-wheel-cache-rp6sqttj/wheels/74/6d/ae/7f4f6994b81c8e10a3ad025eac51f4c4e1dca26ac3eac45ae8
  Building wheel for psutil (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/cindy/Documents/work/syft.js/examples/mnist/syft/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-uizoh7u4
       cwd: /tmp/pip-install-ro7_jxja/psutil/
  Complete output (44 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.6
  creating build/lib.linux-x86_64-3.6/psutil
  copying psutil/_compat.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_psbsd.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_psaix.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_psosx.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_pssunos.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_pslinux.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_psposix.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_common.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/_pswindows.py -> build/lib.linux-x86_64-3.6/psutil
  copying psutil/__init__.py -> build/lib.linux-x86_64-3.6/psutil
  creating build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_aix.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/runner.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_linux.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_unicode.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_system.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_bsd.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_memory_leaks.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_posix.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/__main__.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_osx.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_process.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_contracts.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_misc.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_sunos.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_connections.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/__init__.py -> build/lib.linux-x86_64-3.6/psutil/tests
  copying psutil/tests/test_windows.py -> build/lib.linux-x86_64-3.6/psutil/tests
  running build_ext
  building 'psutil._psutil_linux' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/psutil
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=570 -DPSUTIL_LINUX=1 -I/usr/include/python3.6m -I/home/cindy/Documents/work/syft.js/examples/mnist/syft/include/python3.6m -c psutil/_psutil_common.c -o build/temp.linux-x86_64-3.6/psutil/_psutil_common.o
  psutil/_psutil_common.c:9:10: fatal error: Python.h: No such file or directory
   #include <Python.h>
            ^~~~~~~~~~
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for psutil
  Running setup.py clean for psutil
Successfully built syft
Failed to build psutil
Installing collected packages: psutil, pyzmq, decorator, ipython-genutils, traitlets, jupyter-core, python-dateutil, jupyter-client, entrypoints, pandocfilters, pygments, webencodings, pyparsing, packaging, bleach, pyrsistent, attrs, zipp, importlib-metadata, jsonschema, nbformat, defusedxml, testpath, mistune, nbconvert, ptyprocess, terminado, pexpect, wcwidth, prompt-toolkit, backcall, parso, jedi, pickleshare, ipython, ipykernel, prometheus-client, Send2Trash, notebook, importlib-resources, lz4, syft
    Running setup.py install for psutil ... error
    ERROR: Command errored out with exit status 1:
     command: /home/cindy/Documents/work/syft.js/examples/mnist/syft/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-lv5e2tl4/install-record.txt --single-version-externally-managed --compile --install-headers /home/cindy/Documents/work/syft.js/examples/mnist/syft/include/site/python3.6/psutil
         cwd: /tmp/pip-install-ro7_jxja/psutil/
    Complete output (44 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/psutil
    copying psutil/_compat.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_psbsd.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_psaix.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_psosx.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_pssunos.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_pslinux.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_psposix.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_common.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/_pswindows.py -> build/lib.linux-x86_64-3.6/psutil
    copying psutil/__init__.py -> build/lib.linux-x86_64-3.6/psutil
    creating build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_aix.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/runner.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_linux.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_unicode.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_system.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_bsd.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_memory_leaks.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_posix.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/__main__.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_osx.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_process.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_contracts.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_misc.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_sunos.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_connections.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/__init__.py -> build/lib.linux-x86_64-3.6/psutil/tests
    copying psutil/tests/test_windows.py -> build/lib.linux-x86_64-3.6/psutil/tests
    running build_ext
    building 'psutil._psutil_linux' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/psutil
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=570 -DPSUTIL_LINUX=1 -I/usr/include/python3.6m -I/home/cindy/Documents/work/syft.js/examples/mnist/syft/include/python3.6m -c psutil/_psutil_common.c -o build/temp.linux-x86_64-3.6/psutil/_psutil_common.o
    psutil/_psutil_common.c:9:10: fatal error: Python.h: No such file or directory
     #include <Python.h>
              ^~~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/cindy/Documents/work/syft.js/examples/mnist/syft/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ro7_jxja/psutil/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-lv5e2tl4/install-record.txt --single-version-externally-managed --compile --install-headers /home/cindy/Documents/work/syft.js/examples/mnist/syft/include/site/python3.6/psutil Check the logs for full command output.
```
Running the command `sudo apt-get install python3-dev` allowed me to continue the installation of PySyft. The [resource](https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory) I found may also help others who run into the same issue.

--- a/README.md
+++ b/README.md
@@ -159,7 +159,6 @@ want to put everything under `~/fl-demo` folder.
 
 ### Installation
 
-First, you will need to install following packages.
 It is recommended that you install python packages in separate virtualenv or conda environment, e.g.:
 
 ```bash
@@ -174,60 +173,27 @@ conda create -n syft python=3.7
 conda activate syft
 ```
 
-#### Install PySyft
+Now, you will need to install following packages:
 
-Get the latest `master` branch of PySyft:
+- PySyft. Follow
+  [PySyft installation guide](https://github.com/OpenMined/PySyft/blob/master/INSTALLATION.md)
+  to install the latest `master` branch of PySyft.
 
-```bash
-cd ~/fl-demo
-git clone https://github.com/OpenMined/PySyft
-cd PySyft
-pip install .
-```
+- PyGrid.
+  Follow [PyGrid documentation](https://github.com/OpenMined/PyGrid/#start-grid-platform-locally)
+  to install the latest `dev` branch of PyGrid.
+  NOTE: If you are running PyGrid with docker-compose, you'll need to [build docker images](https://github.com/OpenMined/PyGrid/#13---build-your-own-images-optional)
+  from the latest `dev` branch.
 
-#### Install PyGrid
-
-Get the latest `dev` branch of PyGrid:
-
-```bash
-cd ~/fl-demo
-git clone https://github.com/OpenMined/PyGrid
-cd PyGrid
-git checkout dev
-```
-
-_NOTE_: currently it's required to replace
-
-```text
-syft==0.2.6
-```
-
-with
-
-```text
-git+git://github.com/OpenMined/PySyft#egg=syft
-```
-
-in `pip-dep/requirements.txt` file in PyGrid root folder.
-
-Then continue with install:
-
-```
-pip install .
-```
-
-#### Install Syft.js with MNIST demo
-
-Get the latest `master` branch of syft.js with MNIST demo app included:
-
-```bash
-cd ~/fl-demo
-git clone https://github.com/OpenMined/syft.js
-cd syft.js
-npm install
-cd examples/mnist
-npm install
-```
+- Syft.js with MNIST demo. Check out the latest `master` branch of syft.js with MNIST demo app included:
+  ```bash
+  cd ~/fl-demo
+  git clone https://github.com/OpenMined/syft.js
+  cd syft.js
+  npm install
+  cd examples/mnist
+  npm install
+  ```
 
 #### Seeding the Model & Plan
 
@@ -248,9 +214,9 @@ PyGrid URL will need to be adjusted accordingly in further steps.
 ##### Create Model & Plan
 
 After the PyGrid is running, the next step is to create the model and training plan and host them in the PyGrid.
-PySyft tutorials include MNIST example jupyter notebooks that guide you through this process.
+PySyft tutorials include [MNIST example jupyter notebooks](https://github.com/OpenMined/PySyft/tree/master/examples/experimental/FL%20Training%20Plan) that guide you through this process.
 
-Fire up jupyter notebook:
+Fire up jupyter notebook in PySyft root folder:
 
 ```bash
 cd ~/fl-demo/PySyft

